infra/modules/azure_app_service/tier.tf

```
@@ -0,0 +1,28 @@
+## Tier Definitions
+# Centralized mapping for legacy and new tiers
+
+locals {
+  # Tier mappings for better scalability and readability
+  tier_map = {
+    "test"     = "s"
+    "standard" = "m"
+    "premium"  = "l"
+    "xs"       = "xs"
+    "s"        = "s"
+    "m"        = "m"
+    "l"        = "l"
+    "xl"       = "xl"
+  }
+
+  # Default to the mapped tier
+  tier = local.tier_map[var.tier]
+
+  # SKU name mapping
+  sku_name_mapping = {
+    "xs" = "B1"
```

If everyone agrees, I'd put F1 (very rare but better than two duplicated values)
```suggestion
    "xs" = "F1"
```

---

it is fine for me

---

As I did it is already compatible with the current version, in fact it is not necessary to refactor everything that has already been created with the old form (test, standard and premium).

Regarding the proposal I think it could be a bit confusing, because the association

> s = test
> m = standard
> l = premium

and `xl` was necessary to add a more advanced `sku`, at most if there is no use case we can remove `xs` which is now "extra", but i think that could be useful in the future in my opinion.